spec/app/ics-028-cross-chain-validation/system_model_and_properties.md

```diff
@@ -116,23 +119,32 @@ CCV provides the following system properties.
   - `val` has `Power(cc,hc,val)` voting power on `cc` at height `hc`;
   - `hc'` is the smallest height on `cc` that satisfies `ts(hc') >= ts(hc) + UnbondingPeriod`, i.e., `val` cannot completely unbond on `cc` before `hc'`;   
   - `hp` is the largest height on the provider chain that satisfies `hp << hc`, i.e., `Power(pc,hp,val) = Power(cc,hc,val)`, where `pc` is the provider chain;
-  - `hp'` is the smallest height on the provider chain that satisfies `hc' << hp'`, i.e., `val` cannot completely unbond on the provider chain before `hp'`.  
+  - `hp'` is the smallest height on the provider chain that satisfies `hc' << hp'`, i.e., `val` cannot completely unbond on the provider chain before `hp'`. 
   
   Then for all heights `h` on the provider chain, 
    ```
-  hp <= h < hp': Power(cc,hc,val) <= VP(pBonded(h,val))
+  hp <= h < hp': 
+  Power(cc,hc,val) <= VP( pBonded(h,val) + sum(pUnbonding(hu,val)) + sum(slash(val,hs,hp,_)) ), 
   ```
+  where 
+  - `sum(pUnbonding(hu,val))` is the sum of all tokens `val` start unbonding on the provider at all heights `hu`, such that `hp < hu <= h`;
+  - `sum(slash(val,hs,hp,_))` is the sum of the slashes of `val` at all heights `hs` for infractions committed at `hp`, such that `hp < hs <= h`.
```

Comment:
I think we reach the limits of markdown here ;-)
I had a bit of a hard time understanding that "`sum(pUnbonding(hu,val))` ... where [...] such that `hp < hu <= h`; " would mean in LaTeX sum_{hu=hp+1}^h (pUnbonding(hu,val)).

I wonder whether we should define auxiliary functions `sumUnbonding(h, hp, val)` and `sumSlash(h,hp,val)` before the formula, so that we don't have the open variables hs and hu within the formula.